<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>477</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="profileTab">
     <attribute name="title">
      <string>Профиль</string>
     </attribute>
     <widget class="QLabel" name="helloLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Здравствуйте , Олег</string>
      </property>
     </widget>
     <widget class="QLabel" name="infoLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>У вас 6 непрочитанных сообщений</string>
      </property>
     </widget>
     <widget class="QListWidget" name="messagesList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>491</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(10,100,120,100);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="feedNewsTab">
     <attribute name="title">
      <string>Лента</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="feedsArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>485</width>
           <height>363</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="myNewsFeedTab">
     <attribute name="title">
      <string>Моя лента</string>
     </attribute>
     <widget class="QScrollArea" name="myFeedsArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>487</width>
        <height>365</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>485</width>
         <height>363</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="friendsTab">
     <attribute name="title">
      <string>Друзья</string>
     </attribute>
     <widget class="QListWidget" name="friendsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>321</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="addFriend">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить друга</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteFriend">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:darkred</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:darkred</string>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMessageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить сообщение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createPostButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать пост</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutMeButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Обо мне</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
